blog.html

```diff
@@ -10,16 +10,82 @@
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300italic,300,600,800' rel='stylesheet' type='text/css'>
 
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1>Relaxr</h1>
+				<ul class="headernav">
+					<li class="headerlist">About</li>
+					<li class="headerlist">FAQ</li>
+					<li class="headerlist">Team</li>
+					<li class="headerlist">Contact Us</li>
+					<li class="headerlist">Blog</li>
+				</ul>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+		<div class="clearfix">
+			<main class="clearfix">
+				<div class="rightside">
+					<section class="firstsection">
+						<h2 class="title">How I implemented Relaxr and changed my life in 2 weeks</h2>
+						<img class="image" src="images/blog_photo1.jpg">
+						<p>Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+					<a href="http://www.google.com">Read More</a>
```

Comment:
Careful with indentation here, this `<a>` tag should probably be at the same level as the `<p>` because they are siblings in our DOM tree.